04.class/memoOperation.js

```diff
@@ -0,0 +1,47 @@
+import sqlite3 from "sqlite3";
+
+export const db = new sqlite3.Database("memo.db");
+
+export default class MemoOperation {
+  constructor() {
+    db.run(
+      "CREATE TABLE IF NOT EXISTS memo(id INTEGER PRIMARY KEY AUTOINCREMENT, content TEXT)",
+    );
+  }
+
+  addMemo(memo) {
+    return new Promise((resolve, reject) => {
+      db.run("INSERT INTO memo(content) VALUES(?)", [memo], (err) => {
```

Comment:
```suggestion
      db.run("INSERT INTO memo(content) VALUES(?)", [content], (err) => {
```

content列に保存するのはcontentである、というふうに名前を揃えた方がいいですね
https://bootcamp.fjord.jp/pages/readable-variable-name

Reply:
その通りでした。
memoを追加するという意識が強すぎて、開発者目線に立てていませんでした。
ご指摘ありがとうございます。
修正いたします！